src/OmniSharp.Roslyn.CSharp/Services/Navigation/GoToDefinitionHelpers.cs

```diff
@@ -0,0 +1,51 @@
+﻿#nullable enable
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.FindSymbols;
+using OmniSharp.Extensions;
+using OmniSharp.Options;
+using System.Threading.Tasks;
+
+namespace OmniSharp.Roslyn.CSharp.Services.Navigation
+{
+    internal static class GoToDefinitionHelpers
+    {
+        internal static async Task<ISymbol?> GetDefinitionSymbol(Document document, int line, int column)
+        {
+            var sourceText = await document.GetTextAsync();
+            var position = sourceText.GetPositionFromLineAndOffset(line, column);
```

Comment:
I think we can use this opportunity to clean up a bit - looks like we have two competing extension methods that do the same thing https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp.Roslyn/Extensions/SourceTextExtensions.cs#L10 and https://github.com/OmniSharp/omnisharp-roslyn/blob/aa1018cc281bc8fe2ead8e237b560d15bae1f646/src/OmniSharp.Roslyn/Extensions/TextExtensions.cs#L26-L33

I think we can get rid of one, or am I wrong on something?